### ISSUE TITLE
Already Running

### ISSUE THREAD

**Issue by SaiyanOfDarkness** [TheCheatsrichter/Gw2_Launchbuddy#63]
I keep getting this pop up, even tried running LB as Admin.  Not sure what to do in this case.  Tried clearing gw2cache folders even.

![](https://i.imgur.com/usAG7BY.png)

I get this when selecting 2 accounts and hitting launch.

**Comment by KairuByte** [comment]
That is a Client message, not an LB message. I haven't run into it before. Does this happen only when trying to launch more then one client at a time?

**Comment by SaiyanOfDarkness** [comment]
Only happens when I try more than 1 account at a time, but it doesn't happen all the time.

**Comment by KairuByte** [comment]
I'll be honest, unless we can find a way to reliably reproduce this, it's going to be a tough problem to fix.

I'd hold out for the next "major" version to see if it fixes the problem. I altered the way mutex closing is handled, and the handling of the client start itself. It just might fix the problem.

You could try compiling latest master on your local box to see if it helps anything.

**Comment by KairuByte** [comment]
Reopening, didnt mean to close.

**Comment by KairuByte** [comment]
Have you tried v1.7 by chance? The changes to mutex handling may have corrected the problem you were seeing.

**Comment by SaiyanOfDarkness** [comment]
This was the message produced by LB

![](https://i.imgur.com/W8YtjTh.png)

**Comment by KairuByte** [comment]
Well, (un?)fortunately 1.8 is in the pipeline, and is going to prevent much work on mutex at the moment. We do plan on hitting it for the 1.8 release, and @TheCheatsrichter has an idea on what might be causing some of the problems that I hadn't even considered.

Short term, unfortunately I don't really have much of a solution. The handle closures are using unmanaged code that I scavenged off an old project that's actually no longer available... anywhere as far as I can tell. The original author never replied when I reached out, and there is very little information about closing mutex handles from managed code. I'm not ashamed to say I am out of my element on it, but I'll be damned if we aren't going to get it figured out and working 100%.
